backend/RoomPlanner/room-planner/src/test/java/edu/roomplanner/EncriptionServiceUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package edu.roomplanner;
+
+import edu.roomplanner.service.impl.EncryptionServiceImpl;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class EncriptionServiceUnitTest {
+
+    private EncryptionServiceImpl sut=new EncryptionServiceImpl();
+
+    @Before
+    public void SetUp(){
+        sut.setPassword("key");
+    }
+
+    @Test
+    public void testingDecryption() {
```

Comment:
The method name would be better to be somethig like shouldEncriptThePassword.
Same for the rest of the methods.